dotnet/src/Functions/Functions.Semantic/Extensions/KernelSemanticFunctionExtensions.cs

```diff
@@ -22,6 +23,43 @@ namespace Microsoft.SemanticKernel;
 /// </summary>
 public static class KernelSemanticFunctionExtensions
 {
+    /// <summary>
+    /// Build and register a function in the internal function collection, in a global generic plugin.
+    /// </summary>
+    /// <param name="kernel">Semantic Kernel instance</param>
+    /// <param name="functionName">Name of the semantic function. The name can contain only alphanumeric chars + underscore.</param>
+    /// <param name="functionConfig">Function configuration, e.g. I/O params, AI settings, localization details, etc.</param>
+    /// <returns>A C# function wrapping AI logic, usually defined with natural language</returns>
+    public static ISKFunction RegisterSemanticFunction(
+        this IKernel kernel,
+        string functionName,
+        SemanticFunctionConfig functionConfig)
+    {
+        return kernel.RegisterSemanticFunction("TODO", functionName, functionConfig);
```

Comment:
Is "TODO" expected as the plugin name?

Reply:
I've made `FunctionsCollection.GlobalFunctionsPluginName` public and we use that everywhere.